### PR TITLE
Update datm

### PR DESCRIPTION
**Pull request by DeniseWorthen**
Issue #30 

- Update to current DATM develop

- Hera and Orion logs attached

### CODE REVIEW

**Review by binli2337 — APPROVED** [review]
Looks good.

File: modulefiles/hera.intel/coupled [hunk 1]
```diff
@@ -15,7 +15,8 @@ setenv NCEPLIBS /scratch2/NCEPDEV/nwprod/NCEPLIBS
 ## load contrib environment
 ## load slurm utils (arbitrary.pl  layout.pl)
 ##
-module load contrib sutils
+module use -a /contrib/sutils/modulefiles
+module load sutils
```
Comment (junwang-noaa) [inline modulefiles/hera.intel/coupled]: I thought this change were already committed by Bin, I am curious why this is shown again?

Comment (DeniseWorthen) [reply modulefiles/hera.intel/coupled]: I don't know. I've merged both my develop and this branch w/ that change but for some reason it keeps showing me being one commit behind the develop branch.

**Comment by junwang-noaa** [comment]
It looks to me your branch somehow is not merged with the develop branch.
Did you do "git remote update? It will get you the latest develop branch in
upstream (official repo).

**Comment by DeniseWorthen** [comment]
I'm finally even w/ develop.

**Review by junwang-noaa — APPROVED** [review]

**Pull request merged**